tsconfig.build.json

```diff
@@ -41,7 +41,11 @@
             "accessibility/*": ["tools/accessibility/dist/*"],
             "viewer/*": ["tools/viewer/dist/*"],
             "ktx2decoder/*": ["tools/ktx2Decoder/dist/*"],
-            "vsm": ["tools/vsm/dist/*"]
+            "vsm": ["tools/vsm/dist/*"],
+            "smart-filters": ["dev/smartFilters/dist"],

```

Comment:
Why 3 of these new lines look different? (not ending in /* like all the existing ones)